src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -47,7 +65,7 @@ def create_volume(mgr, volname, placement):
     return create_mds(mgr, volname, placement)
 
 
-def delete_volume(mgr, volname):
+def delete_volume(mgr, volname, metadata_pool, data_pools):
     """
     delete the given module (tear down mds, remove filesystem)
     """
```

Comment:
nit: update the comment mentioning delete pools.

Reply:
Done